Dockerfile-alpine.template

```diff
@@ -43,4 +43,20 @@ RUN adduser -D -u 1000 node \
     && rm -Rf "node-v$NODE_VERSION" \
     && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
+ENV YARN_VERSION 0.0.0
```

Comment:
The issue with the GitHub release being marked as prerelease instead of stable looks like an issue in the Yarn build scripts, as the Windows and Linux releases are built on separate systems and create the GitHub release in different ways. I filed a task to look into it: https://github.com/yarnpkg/yarn/issues/2799